WordPressCom-Stats-iOS/WPStatsService.m

```diff
@@ -227,8 +339,29 @@ - (void)retrieveTodayStatsWithCompletionHandler:(void (^)(StatsSummaryCompletion
         }
     };
     
+    if (!completion) {
+        return;
+    }
+    
+    StatsSummary *summary = [self.ephemory objectForKey:@"TodayStats"];
```

Comment:
This method isn't done yet, more of a stub. I'll definitely make it a constant in that PR :smile: (I was just tired of the warning of the unused failure handler :smiley_cat:)